oars_gb_pkg/helpers/path_planning/short_course_planner.py

```diff
@@ -0,0 +1,145 @@
+'''
+This program implements the short course routing algorithm described in Section 5.1 of Roland Stelzer's Autonomous
+Sailboat Navigation paper. This algorithm uses a polar efficiency function to determine the efficiency of all possible
+angles that a boat can turn, and then compares the efficiencies to determine the optimal boat heading.
+
+Read the paper here: https://www.dora.dmu.ac.uk/bitstream/handle/2086/7364/thesis-optimized-300dpi.pdf
+'''
+import numpy as np
+from turtle import *
+from time import sleep
+
+class ShortCoursePlanner:
+
+    def polar_effeciency(self, windangle, alpha):
+        '''
+        Calculates the speed for a certain angle
+        :param windangle: the wind direction
+        :param alpha: angle being tested (relative to the wind angle)
+        :return: the vector components for the velocity for the angle
+        '''
+        # Convert angle to radians
```

Comment:
You could just require the input to this function be in radians, if you never want to deal with it in degrees. Up to you, but your units should be consistent across all of our code. (It's not really standardized now, so you get to choose. Probably go with whatever your math wants it in.)